src/lib/ReactDataTable/ReactDataTable.tsx

```diff
@@ -40,167 +46,223 @@ const ReactDataTable = <TData,>(props: ReactDataTableProps<TData>) => {
     }
   }`;
 
+  const DraggableRow = ({ row, table }: { row: Row<TData>; table: Table<TData> }) => {
```

Comment:
Could we move this and `TableRows` to a new file?

Reply:
Thanks for the input ✓

`DraggableRows` and `InternalTableRow` are now in a separate file. However, the TableBody itself is still in the ReactDataTable component since it's no longer working if we move it outside. Probably a re-render context issue (?)

Reply:
@neotrow we tried yesterday for a while and the issue seems that as soon as the context is in an external component, it stops working after the first drag.. maybe you have an idea or we can also look into it together 💪
as a future change should not affect the usage of this feature, i will merge this pr as it is now, and if we find a nicer way to split it, we can always do it in a later pr (so @Michele-Masciave can finish his work today)